### PR TITLE
Move andi app into this umbrella project

### PR DESCRIPTION
**Pull request by jdenen**
This PR turns smartcitiesdata into an umbrella project and pulls `smartcitiesdata/andi` in as its first app. There are a bunch of commits and files, but most of them are pulled in via `andi` history. 

Check out the last two commits for things I changed.

As of the opening of this PR (Sept 13, 3:30 PM), this is up-to-date with `andi`'s `master` branch.

### CODE REVIEW

File: apps/tasks/lib/mix/tasks/app/version.ex [hunk 1]
```diff
@@ -0,0 +1,8 @@
+defmodule Mix.Tasks.App.Version do
+  @moduledoc false
+  def run(_) do
+    Mix.Project.config()
+    |> Keyword.get(:version)
+    |> IO.puts()
```
Comment (jeffgrunewald) [inline apps/tasks/lib/mix/tasks/app/version.ex]: Is this worth making a Logger call?

File: apps/tasks/config/config.exs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# This file is responsible for configuring your application
```
Comment (jeffgrunewald) [inline apps/tasks/config/config.exs]: Lotta boiler plate in here.

File: apps/tasks/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Tasks
+
+**TODO: Add description**
```
Comment (jeffgrunewald) [inline apps/tasks/README.md]: What do I do or what am I intended to do?

File: apps/tasks/.formatter.exs [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# Used by "mix format"
```
Comment (jeffgrunewald) [inline apps/tasks/.formatter.exs]: Is this another instance of a file that could inherit from the top-level formatter or do we want to have one per-app?

File: scripts/deploy.sh [hunk 5]
```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
+
+apps=(andi)
```
Comment (LtChae) [inline scripts/deploy.sh]: This releases all the apps every time we tag the mono-repo. Is that intended to be a strategic stop-gap maneuver™ or is this the new state of things going forward?

File: apps/tasks/lib/mix/tasks/app/version.ex (hunk 1 above)
Comment (jdenen) [reply apps/tasks/lib/mix/tasks/app/version.ex]: No, the point of the task is to write the app version to your console. If it put the log level in there too, we'd have to parse that piece.

File: apps/tasks/.gitignore [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# The directory Mix will write compiled artifacts to.
```
[the remark this answers is not in the document]
Comment (jdenen) [reply apps/tasks/.gitignore]: No we don't. I'll fix this.

File: mix.exs [hunk 7]
```diff
@@ -0,0 +1,20 @@
+defmodule Smartcitiesdata.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      apps_path: "apps",
```
[the remark this answers is not in the document]
Comment (jdenen) [reply mix.exs]: Not worth it, IMO.

File: apps/tasks/config/config.exs (hunk 2 above)
Comment (jdenen) [reply apps/tasks/config/config.exs]: 🤷‍♂

File: apps/tasks/README.md (hunk 3 above)
Comment (jdenen) [reply apps/tasks/README.md]: Yeah, yeah. Documentation is important.

File: scripts/deploy.sh (hunk 5 above)
Comment (jdenen) [reply scripts/deploy.sh]: This is the way I intend it to work, but I need to use the strategic stop-gap maneuver™ of per-app versioning in the meantime. So I need to fix that for now.

File: apps/tasks/.formatter.exs (hunk 4 above)
Comment (jdenen) [reply apps/tasks/.formatter.exs]: Top level should be fine. Nice catch.

File: apps/andi/config/config.exs [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+# This configuration file is loaded before any dependency and
+# is restricted to this project.
+
+# General application configuration
+use Mix.Config
+
+# Configures the endpoint
+config :andi, AndiWeb.Endpoint,
+  url: [host: "localhost"],
+  secret_key_base: "z7Iv1RcFiPow+/j3QKYyezhVCleXMuNBmrDO130ddUzysadB1stTt+q0JfIrm/q7",
```
Comment (jeffgrunewald) [inline apps/andi/config/config.exs]: Is this a good time to remove this secret key from the source control before we migrate it into another (more permanent) repo?

Comment (jdenen) [reply apps/andi/config/config.exs]: I think it's best (for local development if nothing else) to leave this here. I put a comment in saying that it should be overwritten.

**Comment by jdenen** [comment]
I completely forgot about how we set that up in `discovery_api`, so that's a good call. Looks like everyone has given the all clear, so I'm going to merge.

**Pull request merged**